source/frontend/src/features/mapSideBar/acquisition/models/PayeeOptionModel.ts

```diff
@@ -102,7 +102,6 @@ export class PayeeOption {
       agreementDate: null,
       expropriationNoticeServedDate: null,
       expropriationVestingDate: null,
-      advancedPaymentServedDate: null,
```

Comment:
hmm, there seems to be some unrelated changes in this PR, thoughts?

Reply:
needed to compile, not sure who's fixing those?